doc/neps/nep-0038-SIMD-optimizations.rst

```diff
@@ -3,10 +3,10 @@ NEP 38 — Using SIMD optimization instructions for performance
 =============================================================
 
 :Author: Sayed Adel, Matti Picus, Ralf Gommers
-:Status: Draft
+:Status: Accepted
 :Type: Standards
 :Created: 2019-11-25
-:Resolution: none
+:Resolution: http://numpy-discussion.10968.n7.nabble.com/NEP-38-Universal-SIMD-intrinsics-td47854.html
```

Comment:
Can you link to the official archive instead? That's more likely to be stable long-term.

Reply:
This one is still linked from https://scipy.org/scipylib/mailing-lists.html for "NumPy-discussion | Read/Search" (https://scipy.org/scipylib/mailing-lists.html is linked from [Contributing to NumPy](https://numpy.org/devdocs/dev/index.html#contributing-to-numpy))  

If we wish to remove nabble it should have a replacement with a better UX than the [archives](https://mail.python.org/pipermail/numpy-discussion/): when a discussion crosses a month, you need to find the rest of it by going to the next month.